debian/control.in

```diff
@@ -45,10 +45,9 @@ Build-Depends: acl-dev,
  ncurses-dev,
  pkg-config,
  po-debconf (>= 0.8.2),
- python-dev,
+# python-dev,
  zlib1g-dev,
  systemd,
- dh-systemd,
  librados-dev <buster> <stretch> <xenial> <bionic>,
  libradosstriper-dev <buster> <stretch> <xenial> <bionic>,
  libcephfs-dev <buster> <stretch> <xenial> <bionic>,
```

Comment:
these conditionals will need to have `<bullseye>` added to work as expected for us